### DISCUSSION TITLE
Rename Username/Password fields on the Rotki startup screen

### DISCUSSION THREAD

**Discussion by A2be** [rotki/rotki#3853] [Ideas]
## Abstract

The current Rotki version (v1.22.2) uses the terminology "Username" / "Password" on the `Sign in` screen.  This is standard app UX for decades in traditional (web2/web1) apps where the model is client/server and the user logs into a centralized server where (much, or all) data resides.

This use of legacy terminology conveys incorrect information to the Rotki user, implying a data model that is both incorrect, and is not the (superior) local and client-controlled data privacy model that Rotki uses.

## Motivation

Better meaning would be conveyed to the end user if the two fields were renamed.  Something like:

- Local ID              (or Database ID)
- DB encryption key

Other better names can be imagined.  But Username/Password conveys the wrong thing, in my view.

## Specification

Just change the names of the two user input fields to something that correctly represents the Rotki private, local, encrypted, user-owned data model.

**Comment by LefterisJP** [comment]
I am going to turn this into a discussion as this is not an actionable issue unless a decision is made.

**Reply by A2be** [reply]
One interesting anecdote:  I was trying to explain Rotki to a tax professional, one who (like most of the ppl doing this work) are used to centralized-server crypto tax accounting software.  He thought this would require his clients to log-in to some software infrastructure on his company-provided client interface.

I was telling him that, no, this isn't that.  I could just send him my Rotki Database, and my Rotki encryption key, and he could have full access to my private/local data (with me having already configured the API keys, etc.) by simply adding my Rotki DB to his computer, and using it via Rotki software, and unencrypting it with my DB encryption key.

**Comment by LefterisJP** [comment]
Thank you for the feedback @A2be.

This is tricky. We definitely have a problem there and there is still, despite the warnings/info boxes, people that confuse the local account with their website account.

Perhaps a simple rename in the UI can help, but not sure on the names.

Local ID and DB encryption key sound indeed much more correct from a technical point of view. But I wonder if a first time user who sees them will understand what they are. They may not know what to do.

A note here is that we also want to add an option to login with ethereum (so you have an account whose local id is an ethereum address and by signing a message with it you also unlock the DB): https://github.com/rotki/rotki/issues/3261

**Comment by yabirgb** [comment]
I see the same problem, people are used so much to the concept of username/password that I believe it will be hard to have anything else. Technically they are good names but for example if my mother get to saw it she wouldn't know what it would mean. Kelsos also redesigned the first account creation path so it could be a good place to educate the users about this terms

**Reply by A2be** [reply]
https://twitter.com/chrishobcroft/status/1478461908424527880

> Chris Hobcraft:  Let's get rid of "username & password" 

Sign in with Ethereum would also be a good way to eliminate the very wrong information that is conveyed to the Rotki user by the old-school terminology "username / password"

**Reply by yabirgb** [reply]
Yes, that is part of https://github.com/rotki/rotki/issues/3261 and would be really cool
